cg/meta/workflow/nf_analysis.py

```diff
@@ -140,7 +140,7 @@ def extract_read_files(
 
     def verify_case_config_file_exists(self, case_id: str, dry_run: bool = False) -> None:
         """Raise an error if config file is not found."""
-        if not dry_run and not Path(self.get_case_config_path(case_id=case_id)).exists():
+        if not dry_run and not Path(self.get_sample_sheet_path(case_id=case_id)).exists():
             raise ValueError(f"No config file found for case {case_id}")
```

Comment:
is this fetching a config or a sample sheet? Try to unify the function names as this can be confusing when debugging

Reply:
a NF sample sheet. Both words `config` and `sample sheet` are broad and depend on context. This is a sample sheet for nf-core/rnafusion: this is, a file specifying sample information. It is not a config in the same sense as for balsamic or mip. It is confusing to call it config in this context since nextflow pipelines also accept a config file that that is a very different file, that's why I renamed it here.

Reply:
Ok, then you could also change the name of `verify_case_config_file_exists` to `verify_case_sample_sheet_exists` maybe?

Reply:
good catch!! YES!! 🌟